.github/workflows/consolidated-ci-cd.yml

```diff
@@ -56,6 +56,23 @@ jobs:
         with:
           fetch-depth: 0
 
+      # Setup Node.js and pnpm for Tailwind CSS build
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'pnpm'
+
+      - name: Install pnpm
+        run: |
+          corepack enable
+          corepack prepare pnpm@8.14.0 --activate
```

Comment:
I've checked the GitHub Actions workflow and it's already set up to run on all platforms (Windows, macOS, and Ubuntu). The Node.js and pnpm setup steps (lines 60-74) are not platform-specific, so Tailwind CSS will be built on all platforms.